core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -106,6 +106,7 @@ private[spark] abstract class BasePythonRunner[IN, OUT](
   protected val envVars: java.util.Map[String, String] = funcs.head.funcs.head.envVars
   protected val pythonExec: String = funcs.head.funcs.head.pythonExec
   protected val pythonVer: String = funcs.head.funcs.head.pythonVer
+  logInfo(s"Python version info: $pythonExec($pythonVer)")
```

Comment:
This is printed multiple times, isn't it?

Reply:
Updated code to print version and package details only once per executor
Added a bit more details to log list of installed packages too. 
A new property `spark.executor.python.worker.log.details` has been added to enable this Python info  logging